### PR TITLE
Support applicationTags.customTags for heartbeat

### PR DESCRIPTION
**Pull request by haosong**
Address #39

### CODE REVIEW

**Comment by ustinov** [comment]
@song-hao make sure you update the version in setup.py to `1.2.1`

File: wavefront_sdk/common/heartbeater_service.py [hunk 1]
```diff
@@ -41,15 +41,16 @@ def __init__(self, wavefront_client, application_tags, components, source):
         if isinstance(components, str):
             components = [components]
         for component in components:
-            self.heartbeat_metric_tags_list.append(
-                {
+            metric_tags = {
                     APPLICATION_TAG_KEY: application_tags.application,
```
Comment (ustinov) [inline wavefront_sdk/common/heartbeater_service.py]: Do we always expect `application_tags` to be not `None`

Comment (song-hao) [reply wavefront_sdk/common/heartbeater_service.py]: yep

**Comment by ustinov** [comment]
@haosong we have to merge this first, before merging https://github.com/wavefrontHQ/wavefront-sdk-python/pull/43

**Pull request merged**